teuthology/task/ceph_ansible.py

```diff
@@ -436,6 +445,9 @@ def run_playbook(self):
             'install',
             run.Raw('setuptools>=11.3'),
             run.Raw(ansible_ver),
+        ])
+        ceph_installer.run(args=[
```

Comment:
this change commit is not needed, please check previous ceph-ansible test results and see what is failing in your runs? , I think the change we need is only related to 'installer.0'